events/schema.py

```diff
@@ -301,6 +312,14 @@ def get_node(cls, info, id):
     def resolve_translations(self, info, **kwargs):
         return self.translations.order_by("language_code")
 
+    def resolve_can_child_enroll(self: EventGroup, info, **kwargs) -> Optional[bool]:
+        child_id = get_node_id_from_global_id(kwargs["child_id"], "ChildNode")
+        try:
+            child = Child.objects.user_can_view(info.context.user).get(id=child_id)
+        except Child.DoesNotExist:
+            return None
```

Comment:
same thing as with `resolve_can_child_enroll`. Is the None there in purpose and thought through?